client/my-sites/stats/stats-purchase/stats-pwyw-uprade-slider/index.tsx

```diff
@@ -0,0 +1,133 @@
+import formatCurrency from '@automattic/format-currency';
+import { useTranslate } from 'i18n-calypso';
+import TierUpgradeSlider from 'calypso/my-sites/stats/stats-purchase/tier-upgrade-slider';
+import { StatsPWYWSliderSettings } from 'calypso/my-sites/stats/stats-purchase/types';
+import './styles.scss';
+
+// TODO: Remove test data.
+// Currently used as a fallback if no plan info is provided.
+// Better approach is to require plan info and do some form of validation on it.
+function getPWYWPlanTiers( minPrice: number, stepPrice: number ) {
```

Comment:
We probably want to remove the test data - I would prefer error rather than wrong data in the case. What do you think?

Reply:
Yes, definitely. We don't want test data showing up in the shipping code. I was thinking we'd remove that once the real data was available. Removing it before makes the set up for testing that little bit harder.

Maybe a smarter approach for leaving the test data in would be to only return it if the feature-flag is enabled. That would prevent it from accidentally showing up in the shipping version.

Reply:
I must have been confused here. The tier data isn't an issue for testing in the PWYW flow. I'll get this removed today as well.